DQMOffline/Trigger/python/BPHMonitor_cff.py

```diff
@@ -3,34 +3,278 @@
 from DQMOffline.Trigger.BPHMonitor_cfi import hltBPHmonitoring
 
 # HLT_PFMETNoMu90_PFMHTNoMu90_IDTight
-Dimuon20_Jpsi_BPHMonitoring = hltBPHmonitoring.clone()
-Dimuon20_Jpsi_BPHMonitoring.FolderName = cms.string('HLT/BPH/DiMu20_Jpsi/')
-Dimuon20_Jpsi_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon20_Jpsi")
-Dimuon20_Jpsi_BPHMonitoring.muoSelection = cms.string("abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
-#muoSelection
-Dimuon10_Jpsi_Barrel_BPHMonitoring = hltBPHmonitoring.clone()
-Dimuon10_Jpsi_Barrel_BPHMonitoring.FolderName = cms.string('HLT/BPH/DiMu10_Jpsi_Barrel/')
-Dimuon10_Jpsi_Barrel_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon10_Jpsi_Barrel")
-Dimuon10_Jpsi_Barrel_BPHMonitoring.muoSelection = cms.string("abs(eta)<1.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
-
-# HLT_PFMETNoMu120_PFMHTNoMu120_IDTight
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring = hltBPHmonitoring.clone()
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.FolderName = cms.string('HLT/BPH/DoubleMu4_3_Jpsi_Displaced/')
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu4_3_Jpsi_Displaced")
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.muoSelection = cms.string("pt>3 & abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
-
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring = hltBPHmonitoring.clone()
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.FolderName = cms.string('HLT/BPH/DoubleMu4_JpsiTrk_Displaced/')
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu4_JpsiTrk_Displaced")
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.muoSelection = cms.string("pt>4 & abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_Jpsi_tnp = hltBPHmonitoring.clone()
+Dimuon0_Jpsi_tnp.FolderName = cms.string('HLT/BPH/DiMu0_Jpsi_L1_NO_OS/')
+Dimuon0_Jpsi_tnp.tnp = cms.int32(1)
+Dimuon0_Jpsi_tnp.nofset = cms.int32(1)
+Dimuon0_Jpsi_tnp.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Jpsi_L1_NoOS")
+Dimuon0_Jpsi_tnp.numGenericTriggerEventPSet.l1Algorithms = cms.vstring("DM0_SQ")
+Dimuon0_Jpsi_tnp.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu7p5_TkX_Jpsi")
```

Comment:
Actually, it was not clear the way I described this method.
"HLT_Mu7p5_TkX_Jpsi" stays for a set of three paths:
* HLT_Mu7p5_Tk2_Jpsi
* HLT_Mu7p5_Tk3p5_Jpsi
* HLT_Mu7p5_Tk7_Jpsi
they are meant to have decreasing prescale values.
For sure we need to monitor the first (Tk2).
If it is not a problem, I would triplicate the module to use each of them.